site/content/posts/2021-06-22-understanding-kubernetes-tests.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Understanding Kubernetes E2E Tests
+image: /img/sonobuoy.svg
+excerpt: Kubernetes testing can be confusing. Let us demystify the test suite for you.
+author_name: John Schnake
+author_url: https://github.com/johnschnake
+categories: [kubernetes, sonobuoy, windows]
+tags: ['Sonobuoy Team', 'John Schnake']
+date: 2021-06-21
+slug: windows-e2e-tests-with-sonobuoy
+---
+
+One of the most common questions that we get from Sonobuoy users is: “Why are so many tests skipped?” Understandably, users want to know that they are running the right set of tests for their Kubernetes clusters but the complexity of the test suite, misnomers, and inaccurate language can make that difficult.
```

Comment:
I _think_ this is correct since I'm ending the sentence with the quoted question. I had to do some searching though for a good explicit source http://guidetogrammar.org/grammar/marks/quotation.htm#:~:text=If%20a%20statement%20ends%20in,do%2C%20who%20preceded%20you%3F%22